internal/output/color.go

```diff
@@ -12,12 +12,62 @@ var colorRx *regexp.Regexp
 
 func init() {
 	var err error
-	colorRx, err = regexp.Compile(`\[(BOLD|UNDERLINE|BLACK|RED|GREEN|YELLOW|BLUE|MAGENTA|CYAN|WHITE|INFO|/RESET)!?\]`)
+	colorRx, err = regexp.Compile(`\[(HEADING|NOTICE|INFO|ERROR|DISABLED|HIGHLIGHT|/RESET)!?\]`)
 	if err != nil {
 		panic(fmt.Sprintf("Could not compile regex: %v", err))
 	}
 }
 
+type ColorTheme interface {
+	Heading(writer io.Writer)
+	Notice(writer io.Writer)
+	Info(writer io.Writer)
+	Error(writer io.Writer)
+	Disabled(writer io.Writer)
+	Highlight(writer io.Writer)
+	Reset(writer io.Writer)
+}
+
+type defaultColorTheme struct{}
+
+// Heading switches to bold and bright foreground
+func (dct defaultColorTheme) Heading(writer io.Writer) {
+	ct.Foreground(writer, ct.White, true)
+	ct.ChangeStyle(writer, ct.Bold)
+}
+
+// Notice switches to bright foreground
+func (dct defaultColorTheme) Notice(writer io.Writer) {
+	ct.Foreground(writer, ct.White, true)
```

Comment:
Isn't this also setting it as white? The AC just calls for "bright".

Reply:
Unfortunately, it is not possible to just set the "bright" flag, you always have to specify a color, and white is the default color.  When I set "white+bright" on my terminal which has a solarized color theme, it actually shows up as bright black (or grey-ish).  So, I think that's the best we can do here.

Reply:
I see. In that case let's merge `HIGHLIGHT` into `NOTICE`. So let's drop `HIGHLIGHT` and instead make all those instances use `NOTICE`.

Reply:
Just to make sure that I undestand correctly:  Would you like to remove the HIGHLIGHT formatter completely (although it is using a different color `teal`)?

I exchanged some uses of `[BOLD] a url or something [/RESET]` with highlight which seems sensible to me.  Do you want me to change these to `[NOTICE]` or remove them completely?

Reply:
Mostly `[HIGHLIGHT]` is currently used for State Tool commands that could be copied and pasted to the command line.

Reply:
Ok, let's maybe just make the difference more distinct then. Let's rename `HIGHLIGHT` to `ACTIONABLE`.